tests/propertyBinding/impl/cpp/PropertyBindingInterfaceCppImplementation.h

```diff
@@ -29,21 +29,26 @@
 **********************************************************************/
 
 #pragma once
-#include "tests/propertybinding/PropertyBindingInterfaceTestImplementationBase.h"
+#include "tests/propertybinding/PropertyBindingInterfaceTestPropertyAdapter.h"
```

Comment:
??

Reply:
@weimerb  Can answer here?

Reply:
Why would you change it back? Don't use the deprecated include.

Reply:
ok , reverted back.
Thanks